hilda/ipython_extensions/events.py

```diff
@@ -0,0 +1,48 @@
+import ast
+import builtins
+
+import lldb
+
+from hilda.exceptions import SymbolAbsentError
+from hilda.symbols_jar import SymbolsJar
+
+
+class HIEvents:
+    def __init__(self, ip):
+        self.shell = ip
+        self.hilda_client = self.shell.user_ns['p']
+
+    def pre_run_cell(self, info):
+        """
+        Enable lazy loading for symbols
+        :param info: IPython's CellInfo object
+        """
+        if info.raw_cell[0] in ['!', '%'] or info.raw_cell.endswith('?'):
```

Comment:
stop is needed for that. avoid this entirely if it isn't, write a warning instead

Reply:
TODO comment still remains